src/APIGateway/APIGateway/ocelot.json

```diff
@@ -0,0 +1,93 @@
+{
+  "Routes": [
+    {
+      "DownstreamPathTemplate": "/items",
+      "DownstreamScheme": "http",
+      "DownstreamHostAndPorts": [
+        {
+          "Host": "localhost",
+          "Port": 5002
+        }
+      ],
+      "UpstreamPathTemplate": "/api/catalog/items",
+      "UpstreamHttpMethod": [ "GET" ],
+      "Key": "Items"
+    },
+    {
+      "DownstreamPathTemplate": "/items/1/properties",
+      "DownstreamScheme": "http",
+      "DownstreamHostAndPorts": [
+        {
+          "Host": "localhost",
+          "Port": 5002
+        }
+      ],
+      "UpstreamPathTemplate": "/api/catalog/items/{id}/properties",
+      "UpstreamHttpMethod": [ "GET" ],
+      "Key": "ItemProperties"
+    },
+    {
+      "DownstreamPathTemplate": "/{everything}",
+      "DownstreamScheme": "http",
+      "DownstreamHostAndPorts": [
+        {
+          "Host": "localhost",
+          "Port": 5002
+        }
+      ],
+      "FileCacheOptions": {
+        "TtlSeconds": 60
+      },
+      "UpstreamPathTemplate": "/api/catalog/{everything}",
+      "UpstreamHttpMethod": [ "GET" ]
+    },
+    {
+      "DownstreamPathTemplate": "/{everything}",
+      "DownstreamScheme": "http",
+      "DownstreamHostAndPorts": [
+        {
+          "Host": "localhost",
+          "Port": 5002
+        }
+      ],
+      "UpstreamPathTemplate": "/api/catalog/{everything}",
+      "UpstreamHttpMethod": [ "POST", "DELETE", "PATCH" ],
+      "AuthenticationOptions": {
+        "AuthenticationProviderKey": "Bearer",
+        "AllowedScopes": [
+          "catalog:items:create",
+          "catalog:categories:read",
+          "catalog:items:read",
+          "catalog:categories:update",
+          "catalog:categories:create",
+          "catalog:items:delete",
+          "catalog:items:update",
+          "catalog:categories:delete"
+        ],
+        "AllowedRoles": [ "manager" ]

```

Comment:
Good, `manager` roles is added, and scopes look valid, however there's a specific role needed `admin` it can be just a clone of `manager` role.
Catalog service which mutates data (POST/PUT/PATCH/DELETE) should be accessible by the admins `(Role=admin)` only